umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif9/spatial_extent.clj

```diff
@@ -0,0 +1,72 @@
+(ns cmr.umm-spec.umm-to-xml-mappings.dif9.spatial-extent
+  "These are functions that create a data structure that
+   will make it easier to perform the vertical spatial translation
+   for UMM -> DIF 9 records.")
+
+(defn create-elevation-key
+  "This function looks at the passed in type and checks to see if the tye contains the values
+   of Altitude with Min or Max or Depth with Min or Max. If any of these match then I can
+   match the type with the DIF 9 key. Otherwise pass back nil."
+  [elevation-type]
+  (if (.contains elevation-type "Altitude")
+    (if (.contains elevation-type "Min")
+      (keyword (str "Minimum_Altitude"))
+      (if (.contains elevation-type "Max")
+        (keyword (str "Maximum_Altitude"))))
+    (if (.contains elevation-type "Depth")
+      (if (.contains elevation-type "Min")
+        (keyword (str "Minimum_Depth"))
+        (if (.contains elevation-type "Max")
+          (keyword (str "Maximum_Depth")))))))
+
+(defn- create-vertical-domain-maps
+  "Returns a vector that contains a number of maps with vertical domain data in it.
```

Comment:
I had a hard time understanding this from the docstring, but when I looked at the tests I got a much better idea of what was going on. Can you use a paired down example from the test in the docstring? Like with X input I get Y result. That way we could see both the format and what the function is actually doing? Otherwise the dosctring and comments are really good.